testsuite/features/build_validation/retail/sle12sp5_buildhost_build_kiwi_image.feature

```diff
@@ -1,6 +1,12 @@
 # Copyright (c) 2021-2024 SUSE LLC
 # Licensed under the terms of the MIT license.
 
+# TODO: This feature is not working within a proxy containerized environment
+#       due to the fact that the mgr-bootstrap command is not available in the proxy
+#       container. The issue still under research.
+
+@skip_if_container_server
+@skip_if_github_validation
```

Comment:
Instead of writing `TODO` with a long-format comment multiple times, I would leave just one line comment in each section referencing the issue that would address these problems.

Reply:
We still don't have a card about it, but from what I understood from @aaannz it might be just something to refactor in our step definition. For now, if you don't mind, I will let it just like that.

Reply:
You can create the card right now remarking else need to be done aside from this PR with regard to this topic. However, if you prefer to leave it like that I don't mind.

Reply:
Still an open discussion here https://suse.slack.com/archives/C02CLB6AGJ3/p1709221802923919 I don't even know if we will need a card or it's a product enhancement.